examples/local_debugging/README.md

```diff
@@ -1,6 +1,14 @@
 # Local debugging
 
-This example shows how to run the [Titanic example](../titanic/README.md) locally to debug your code.
+- [Local debugging](#local-debugging)
+  - [Prerequisites](#prerequisites)
+  - [Data and script preparation](#data-and-script-preparation)
+  - [Run and debug our pipeline](#run-and-debug-our-pipeline)
+  - [Difference with the Titanic example](#difference-with-the-titanic-example)
+- [Hybrid debugging](#hybrid-debugging)
+
+This example shows how to run the [Titanic example](../titanic/README.md) locally to debug your code. There are two possibilities:
+either run it completely locally or use the fake data from the dataset from the deployed Substra platform with a local client.
```

Comment:
The difference between the two possibilities may be obscure to most users, what do you think of:

```md
There are two possibilities as to which data to use for debugging:
* if you have data samples available on disk: use those
* if you don't: use fake data samples containing random data generated on the fly by the opener
```

Reply:
Better ! I would only change
> use fake data samples containing random data generated on the fly by the opener

to

> use random data generated on the fly by the opener

because there are no data samples created.

Reply:
Very good point!